index.html

```diff
@@ -49,10 +76,209 @@ <h2 class="CONNECT"><a href="#">LET'S CONNECT</a> </h2>
+          
+      
+      
+              
+          <div class="p-div" >
+              <p class="line">I’m a software developer! I can help you build a product , feature or website Look through <br> 
+                  some of my work and experience! If you like what you see and have a project you need <br>
+                  coded, don’t hestiate to contact me</p>
+          </div>
+          
+                  <div class="div-CONNECT2" >
+                      <h2 class="CONNECT2"><a href="#">LET'S CONNECT</a> </h2>
+                  </div>
+          
+      
+      
+                <div>
+      
+                  <ul class="fab-ba">
+                   <li><a href="https://twitter.com/zoumaniguimass1" target="_blank"><i class="fab fa-twitter"></i></a></li>
+                     <li> <a href="#" target="_blank"><i class="fab fa-facebook"></i></a></li>
+                     <li>  <a href="https://github.com/massazoum" target="_blank"><i class="fab fa-github" aria-hidden="true"></i></a></li>
+                     <li>  <a href="" target="_blank"><i class="fab fa-linkedin" aria-hidden="true"></i></a></li>
+          
+                  </ul>
+      
+                 </div>
+                  <div class="bnt-div"><button class="bnt-get" type="button">Get my resume</button></div>
+             </div>
+          
+      
+      
+          
+                            <div class="second-about">
+                                  <div class="languages2">
+                                      <h1><div>languages</div><div class="ico1"><img src="chevron-down.png" width="70%" height="80%" alt=""></div></h1>
+                                      <ul>
+                                          <li class="list"><div class="ico"><img src="jsIco.png" width="100%" height="100%" alt=""></div><div>JavaScript</div></li>
+                                          <li class="list"><div class="ico"><img src="htmlIco.png" width="100%" height="100%" alt=""></div><div>html</div></li>
+                                          <li class="list"><div class="ico"><img src="cssIco.png" width="100%" height="100%" alt=""></div><div>css</div></li>
+                                  </div>
+                                  <div class="framework">
+                                      <ul>
+                                          <li class="list1"><div>Framework</div><div class="ico1"><img src="chevron-right.png" width="80%" height="70%" alt=""></div></li>
+                                      </ul>
+                                          <hr/>
+                                      <ul>    
+                                          <li class="list1"><div>skills</div><div  class="ico1"><img src="chevron-right.png" width="80%" height="70%" alt=""></div> </li>
+                                      </ul> 
+                                      <hr/>
+                                  </div>
+                              </div>
+                    <div>
```

Comment:
Kindly also update the styling of your languages  section to better match the figma design🙏.
Your design:
<img width="234" alt="image" src="https://user-images.githubusercontent.com/37116322/220424901-8e72c35b-d237-4265-9fc8-d95d96178618.png">

Figma design:
<img width="221" alt="image" src="https://user-images.githubusercontent.com/37116322/219809809-538619ce-e978-49ed-9b29-59c28eeab244.png">

A few things to note:
- Each of the language section titles should be aligned to the far left and the arrows aligned to the far right
- The technologies(HTML, CSS, JS) should also be aligned to the far right and be on the same line with the languages section title

**Please try your best to make sure that your design matches the Figma design as much as possible😎**